### PR TITLE
hw03

### PR DESCRIPTION
**Pull request by lendijoy**
@diegodulanto 
@sejalkesh

### CODE REVIEW

File: hw03/hw03.Rmd [hunk 1]
```diff
@@ -21,10 +21,30 @@ For your assignment, do the following.
...
+```
+
   2. Compute total scores for the four scales. 
      Recode variables as needed.
+```{r}
+reverse <- function(x) (6 - x)
+green_data_comp <- green_data %>% 
+   rowwise() %>% 
+  mutate(across(c(green2, comp6:comp10, intel7:intel10, open7:open10), reverse)) %>% 
+  mutate(green_mean = mean(c_across(starts_with("g")), na.rm = TRUE), comp_mean = mean(c_across(starts_with("c")), na.rm = TRUE), intel_mean = mean(c_across(intel1:intel10), na.rm = TRUE), open_mean = mean(c_across(starts_with("o")), na.rm = TRUE)) %>%
```
Comment (diegodulanto) [inline hw03/hw03.Rmd:45]: I recommend spacing out your code onto new lines!

**Review by diegodulanto — COMMENTED** [review]
Great work overall!

File: hw03/hw03.Rmd [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+summary(green_data)
+```
+
+  2. Compute total scores for the four scales. 
+     Recode variables as needed.
+```{r}
+reverse <- function(x) (6 - x)
+green_data_comp <- green_data %>% 
+   rowwise() %>% 
+  mutate(across(c(green2, comp6:comp10, intel7:intel10, open7:open10), reverse)) %>%
```
Comment (sejalkesh) [inline hw03/hw03.Rmd:44]: Don't forget to indent and space your code out especially when using multiple functions.

File: hw03/hw03.Rmd [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+For your assignment, do the following.
+
+  1. Inspect the item responses (e.g., with graphs or by summarizing distinct values). 
+     Is anything unusual?
+```{r}
+summary(green_data)
+```
+<<<<<<< HEAD
+Several variables appear to have missing values coded as -99, so we will remove those.
```
Comment (sejalkesh) [inline hw03/hw03.Rmd:31]: Good job explaining the steps and what the code is meant to do!

**Pull request merged**